src/bridging/providers/across/createAcrossDepositCall.test.ts

```
@@ -0,0 +1,193 @@
+          ...acrossQuote,
+          fromChainId: invalidChainId,
+        },
+        cowShedSdk: cowShedSdk,
+      })
+    ).toThrow('Spoke pool address not found for chain: ' + invalidChainId)
+  })
+
+  it('should throw error for invalid math contract address', () => {
+    // Temporarily modify ACROSS_MATH_CONTRACT_ADDRESSES to simulate missing math contract
+    const originalMathAddresses = { ...ACROSS_MATH_CONTRACT_ADDRESSES }
+    delete ACROSS_MATH_CONTRACT_ADDRESSES[1]
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)